ml-agents/mlagents/trainers/trajectory.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, NamedTuple
+import numpy as np
+
+from mlagents.trainers.buffer import AgentBuffer
+
+
+class AgentExperience(NamedTuple):
+    obs: List[np.ndarray]
+    reward: float
+    done: bool
+    action: np.ndarray
+    action_probs: np.ndarray
+    action_pre: np.ndarray  # TODO: Remove this
+    action_mask: np.ndarray
+    prev_action: np.ndarray
+    max_step: bool
```

Comment:
nit: `is_max_step` (`max_step` sounds like an int)

Reply:
Keeping symmetry with the BatchedState (which called it `max_step`)